lib/smart_answer_flows/benefit-cap-calculator/questions/enter_postcode.govspeak.erb

```diff
@@ -1,8 +1,12 @@
 <% content_for :title do %>
-  Enter your postcode:
+  Enter your postcode
 
 <% end %>
 
+<% content_for :body do %>
+  eg SW1A 2AA
+<% end %>
+
```

Comment:
I agree and did consider splitting it. I decided against it partly due to time available and partly as I wanted to reuse as many commits as possible from PR #2542.